packages/app/src/runner/event-manager.ts

```diff
@@ -540,6 +540,18 @@ export class EventManager {
       })
     })
 
+    Cypress.on('run:start', async () => {
+      if (Cypress.config('experimentalMemoryManagement') && Cypress.isBrowser({ family: 'chromium' })) {
+        await Cypress.backend('start:memory:profiling', Cypress.config('spec'))
```

Comment:
If this fails, we aren't logging anything to indicate something when wrong.Seems like something we should add server-side to ensure the full picture is available in the logs. The error is being swallowed here

Reply:
There is a [debug](https://github.com/cypress-io/cypress/blob/mschile/chrome_memory_fix/packages/server/lib/browsers/memory/index.ts#L341) log that gets generated if it fails to start. Are you looking for something else?

Reply:
I guess this partially covers its - this try-catch is partially around the startProfiling implementation. There isn't any under endProfiling or maybeCollectMemory profiling though at the top level to ensure we don't accidentally bonk out.

Reply:
Saving the file is the only thing that can fail in `endProfiling` and that is handled [here](https://github.com/cypress-io/cypress/blob/mschile/chrome_memory_fix/packages/server/lib/browsers/memory/index.ts#L354). I also catch potential errors when gathering the memory stats [here](https://github.com/cypress-io/cypress/blob/mschile/chrome_memory_fix/packages/server/lib/browsers/memory/index.ts#L289). I'll add something to `maybeCollectGarbage`.

Reply:
updated [0a00475](https://github.com/cypress-io/cypress/pull/25462/commits/0a004751178122988dcc43ef6fecf70d9ae23dc7)